registry/handlers/app.go

```diff
@@ -274,6 +276,28 @@ func (app *App) configureRedis(configuration *configuration.Configuration) {
 	}))
 }
 
+// configureMail prepares logging hook parameters.
+func (app *App) configureLogHook(configuration *configuration.Configuration) {
+	logger := ctxu.GetLogger(app).(*log.Entry).Logger //Type: *logrus.Logger
+
+	hook := &LogHook{}
+	hook.LevelsParam = configuration.Log.Hooks.Level
+	hook.Type = configuration.Log.Hooks.Type
+	if hook.Type == "email" {
```

Comment:
It is more in the style of go to return early on error: in this case if configuration.Log.Hooks.Type != "email", you can just return from the function.  You should also log if there is an unsupported hook type configured